sematic/examples/lightning_resnet/pipeline.py

```diff
@@ -0,0 +1,162 @@
+# Standard Library
+import logging
+from dataclasses import dataclass
+
+# Third-party
+import ray
+
+# Sematic
+import sematic
+from sematic import context
+from sematic.ee.ray import RayCluster, SimpleRayCluster
+from sematic.examples.lightning_resnet.checkpointing import (
+    Checkpoint,
+    SematicCheckpointIO,
+)
+from sematic.examples.lightning_resnet.train_eval import (
+    DataConfig,
+    EvaluationConfig,
+    EvaluationResults,
+    TrainingConfig,
+    evaluate_classifier,
+    train_classifier,
+)
+
+
+@dataclass
+class PipelineResults:
+    """Data structure for holding results from training and evaluation"""
+
+    final_checkpoint: Checkpoint
+    evaluation_results: EvaluationResults
+
+
+@sematic.func(inline=False)
+def train(config: TrainingConfig, data_config: DataConfig) -> Checkpoint:
+    """# Perform distributed training of a ResNet model.
```

Comment:
```suggestion
    """# Perform distributed training of a ResNet model
```

Reply:
This is a section header, which usually doesn't include periods

Reply:
Yes, the suggestion is to remove the period. Please do so everywhere.